src/core/annotation.js

```diff
@@ -1961,8 +1999,12 @@ class WidgetAnnotation extends Annotation {
     }
 
     let lineCount = -1;
+    let lines;
     if (this.data.multiLine) {
-      lineCount = value.split(/\r\n|\r|\n/).length;
+      lines = value.split(/\r\n|\r|\n/).map(line => line.normalize("NFKC"));
```

Comment:
Here, and below, we could probably use the slightly shorter `/\r\n?|\n/` regular expression instead (as we do elsewhere in the code-base).

Also, can you please add a brief comment explaining what that particular normalization mode does and why it was chosen here?